lib/game.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'player'
+class Game
+  attr_reader :player_name, :player_choice
+  def initialize(player_choice, player_name = Player.new)
+    @player_name = player_name
+    @player_choice = player_choice
+    @options = ["rock", "paper", "scissors"]
+  end
+
+  def computer_choice
+    @options[rand(2)]
+  end
+
+  def round_player_rock
+    case 
+      when @player_choice == "rock" && computer_choice == "rock" then "Draw!"
+      when @player_choice == "rock" && computer_choice == "scissors" then "You won!"
+      when @player_choice == "rock" && computer_choice == "paper" then "You lost!" 
+    end
+  end
+
+  def round_player_paper
+    case 
+      when @player_choice == "paper" && computer_choice == "paper" then "Draw!"
```

Comment:
You could try these options to keep your code dry:

1. for all the "Draw!" options, group them into `when @player_choice == computer_choice` then "Draw!"